index.html

```diff
@@ -4056,9 +4106,9 @@ <h3>Reserved Extension Points</h3>
               <td>`confidenceMethod`</td>
               <td>
 A property used for specifying one or more methods that a verifier might use to
-increase their confidence that the value of an attribute in or of a verifiable
+increase their confidence that the value of a property in or of a verifiable
 credential or verifiable presentation is accurate, including but not limited to
-attributes such as `initialRecipient` (a/k/a `issuee`), `presenter`,
+properties such as `initialRecipient` (a/k/a `issuee`), `presenter`,
 `authorizedPresenter`, `holder`, etc. The associated vocabulary URL MUST be
```

Comment:
Here we have ``` `initialRecipient` (a/k/a `issuee`) ``` as well as ``` `authorizedPresenter` ```.

It seems like these should perhaps be added to the VCDMv2 reserved terms list.

I am also concerned that `confidenceMethod` is itself severely [underspecified](https://w3c-ccg.github.io/confidence-method-spec/#confidence-method), and does not appear to discuss any of the above properties.

Reply:
(Editor hat on) The description of confidence method is problematic. Given that almost no work has gone into it since it was introduced, I'm hesitant to reserve properties that it contemplates (that aren't documented at all).

(Digital Bazaar hat on) We have found use cases where confidence method is useful, so expect it to be more fully formed in time. However, for the time being, I'd rather not do anything about this issue in this PR.

We might want to raise another issue about `confidenceMethod`... or just deal with it when we have to make a judgement call on reserving it or not.

Reply:
I agree with @msporny that this should be dealt with in a separate issue.